conda/cli/conda_argparse.py

```diff
@@ -115,8 +116,11 @@ def do_call(args, parser):
     """
     # First, check if this is a plugin subcommand; if this attribute is present then it is
     if getattr(args, "plugin_subcommand", None):
-        _run_pre_command_hooks(args.plugin_subcommand.name, args)
-        return args.plugin_subcommand.action(sys.argv[2:])
+        _run_command_hooks("pre", args.plugin_subcommand.name, sys.argv[2:])
+        result = args.plugin_subcommand.action(sys.argv[2:])
+        _run_command_hooks("post", args.plugin_subcommand.name, sys.argv[2:], result)
```

Comment:
I wonder if we should do some error handling here, following #12149 this seems prudent?
```suggestion
        try:
            _run_command_hooks("post", args.plugin_subcommand.name, sys.argv[2:], result)
        except Exception as exc:
            do_something_with_it(exc)
        finally:
            return result
            
```

Reply:
Is this because whatever happened in the command is assumed to have succeeded at this point? (otherwise an exception would have been thrown), and therefore, we don't want to let an uncaught error blow up in a big stack trace?

Something similar came up here: https://github.com/conda/conda/pull/12712#discussion_r1205679348

I'd be more than happy to implement what I originally had planned for `pre_command` for `post_command`.

Ultimately, the plugin should be held responsible for catching and handling all exceptions, but if one falls through the cracks, we could tell users about it and how they might fix the situation (e.g. remove the plugin or file a bug issue with the plugin author).